docker/erobs-common-img/README.md

```diff
@@ -23,27 +23,35 @@ export GHCR_POINTER=ghcr.io/chandimafernando/erobs-common-img:latest
 
 # Enable the connection between the UR robot and the VM
 ```bash
-podman run -it --network host --ipc=host --pid=host \
+podman run -it --rm --network host --ipc=host --pid=host \
     --env ROBOT_IP=$ROBOT_IP \
     --env UR_TYPE=$UR_TYPE \
     --env ROS_DISTRO=$ROS_DISTRO \
     --env DESCRIPTION_PKG=$DESCRIPTION_PKG \
     --env DESCRIPTION_FILE=$DESCRIPTION_FILE \
     ${GHCR_POINTER} \
-    /bin/sh -c "printenv && . /opt/ros/${ROS_DISTRO}/setup.sh && . /root/ws/install/setup.sh && ros2 launch ur_robot_driver ur_control.launch.py ur_type:=${UR_TYPE} robot_ip:=${ROBOT_IP} description_package:=${DESCRIPTION_PKG} description_file:=${DESCRIPTION_FILE} launch_rviz:=false tool_voltage:=24"
+    /bin/sh -c "printenv && . /opt/ros/${ROS_DISTRO}/setup.sh && . /root/ws/install/setup.sh && ros2 launch ur_robot_driver ur_control.launch.py ur_type:=${UR_TYPE} robot_ip:=${ROBOT_IP} description_package:=${DESCRIPTION_PKG} description_file:=${DESCRIPTION_FILE} launch_rviz:=${LAUNCH_RVIZ} tool_voltage:=24"
 ```
 
 # Run gripper service to enable the gripper 
 ```bash
-podman run -it --network host --ipc=host --pid=host \
+podman run -it --rm --network host --ipc=host --pid=host \
+    --env ROBOT_IP=$ROBOT_IP \
+    --env ROS_DISTRO=$ROS_DISTRO  \
     ${GHCR_POINTER} \
     /bin/bash -c ". /root/ws/install/setup.sh && \
+    . /opt/ros/${ROS_DISTRO}/setup.sh && \
+    ros2 run ur_robot_driver tool_communication.py --ros-args -p robot_ip:=${ROBOT_IP} & \
+    sleep 5 && \
+    . /root/ws/install/setup.sh && \
     ros2 run gripper_service gripper_service"
 ```
+Note: tool_communication.py should not be called when ur_control.launch.py is called, but separately when the gripper is initiated. 5s delay allows socat creation before running the gripper service.
+Note: gripper_service has two nodes, gripper_open and gripper_close to utilize if needed.
```

Comment:
```suggestion
---
**NOTE**

`tool_communication.py` should not be called when `ur_control.launch.py` is called, but separately when the gripper is initiated.  The 5s delay allows socat creation before running the gripper service. 
The gripper_service has two convenience scripts, `gripper_open.cpp` and `gripper_close.cpp` that will start up client nodes to send a request to the gripper service then promptly shutdown the client nodes.

---

Reply:
Readme is updated:
https://github.com/maffettone/erobs/blob/5c32dc6fe6c0a002d6ecb2d3e5b09daddd49ae45/docker/erobs-common-img/README.md?plain=1#L52-L53